# CustomChannelImage.msg
float32[] data
uint32 height
uint32 width
uint32 num_channels